cordcutter/__init__.py

```diff
@@ -63,6 +66,13 @@ async def _on_application_command_error(
         if self.errors.get(interaction.application_command, 0) >= self.threshold:
             return
 
+        if self.ignore_exceptions:
+            for ignore_exception in self.ignore_exceptions:
+                if isinstance(exception, ignore_exception):
+                    return
+                elif ignore_exception.__name__ in str(exception):
+                    return
```

Comment:
This is a bit weird... What's the purpose of it?

Reply:
Okay, the current implementation is a bit bad, I decided to improve it in the new commit.

As for your question - it all boils down to the fact that nextcord returns an `ApplicationInvokeError` exception if the error is built in Python. For example, if the command calls RuntimeError, nextcord will return ApplicationInvokeError in the method anyway. Therefore, if one gives RuntimeError for ignored exceptions then it won't work. Previously I didn't have much of an idea how to fix it, but now I've improved it a bit and you can let me know what you think.

Reply:
You can probably do something like the following:
```py
error = getattr(error, "original", error)

if isinstance(error, RuntimeError):
    ...

...
```
Or an if-statement checking for ApplicationInvokeError and then assigning error to `.original`